build.gradle

```diff
@@ -24,3 +24,10 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+ext {
+    supportLibVersion = "26.1.0"
```

Comment:
I don't prefer this arrangement in jumpstart project. It will make updating dependencies harder in future. Thoughts?